### ISSUE TITLE
Support other logging URLs besides Kibana

### ISSUE THREAD

**Issue by fhemberger** [whs/kube-slack#4]
It would be nice to customize the logging url for other services than Kibana. Maybe by adding `LOGGING_URL` with some kind of placeholders for `container` and `pod` to be replaced by a RegExp?

Example:
```js
let loggingUrl = '';
if (process.env.LOGGING_URL) {
  loggingUrl = process.env.KIBANA_URL
    .replace('%%pod%%', encodeURIComponent(item.pod.metadata.name))
    .replace('%%container%%', encodeURIComponent(item.name));

  loggingUrl = `(<${loggingUrl}|View logs>)`;
}
```

**Comment by whs** [comment]
Fixed in 08afa3a840429b083afc6100150b3e9c66034344 . If you're running with `latest` tag please see the update instruction in readme. (Although I would recommend pinning your version instead)

**Comment by fhemberger** [comment]
Cool, thank you!
